miditok/midi_tokenizer.py

```diff
@@ -1458,6 +1471,7 @@ def _create_track_events(
 
             # Duration / NoteOff
             if use_durations:
+                #TODO: Should this first part be in the duration fn?
```

Comment:
Can it be removed now?

Reply:
Are you referring to the new '_create_duration_event()' function a few lines down? If so, this is still very much necessary, as MidiTok's way of creating duration events is incompatible with PerTok at the moment, so I still need a way to override this

Reply:
Was just checking :)